frontend/src/App.tsx

```diff
@@ -4,12 +4,14 @@ import React from 'react';
 import Home from './pages/home';
 import { Route, Routes } from 'react-router-dom';
 import DevPage from './pages/dev';
+import Recommendations from './pages/recommendations';
 
 function App() {
   return (
     <Routes>
       <Route path="/dev" element={<DevPage />} />
       <Route path="/" element={<Home />} />
+      <Route path="/" element={<Recommendations />} />
```

Comment:
Can you change the path to "/recommendations" and also put this at the top of the Routes?

Reply:
Otherwise I think you won't be able to see recommendations page at all because of the overlap in paths